lib/backend/fetch.dart

```diff
@@ -33,6 +24,14 @@ class CancelableFsFetch {
     this.sortType = SortType.name,
     this.showHidden = false,
   });
+  final Directory directory;
```

Comment:
telling you here to avoid repeating it every single time, wouldn't it be better to have an empty new line between constructor and fields? like this i find it a bit ugly, also why moving the fields after the constructor? stylistic choice?

Reply:
what I did was lint rules, updated flutter_lints and dart fix --apply and dart format . 

:)

if you disagree with this rule we can change it the way you want
just thought would be good to have a consistency

Reply:
I think I'd prefer the way it was before, could be discussed more with other people tho

Reply:
reverted and revoved the lint rule